skytemple_files/common/ppmdu_config/pmdsky_debug/test/migration_test.py

```diff
@@ -0,0 +1,243 @@
+#  Copyright 2020-2022 Capypara and the SkyTemple Contributors
```

Comment:
In addition to these tests, could it be useful to add a snapshot/diff test for currently defined symbols? That might make it easier to tell what's changed when updating the submodule.

Reply:
See above.